components/tabs/stories/tabs.stories.js

```diff
@@ -1,10 +1,24 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { version } from "../package.json";
 import { TabsGroups } from "./tabs.test.js";
-import { Template } from "./template.js";
+import { Template, QuietGroup, OverflowGroup, VerticalGroup, CompactGroup } from "./template.js";
 
 /**
  * Tabs organize content into multiple sections and allow users to navigate between them. The content under the set of tabs should be related and form a coherent unit.
+ * 
+ * ## Use icons consistently
+ * 
+ * Icons are optional, but don’t mix the use of icons in tabs if they are utilized. Navigation controls require a clear spacial relationship to one another, and mixing the use of icons can dramatically impact the visual balance and presence for each tab item.
+ * 
+ * ## Selected item
+ * Only one tab item can be selected at any given time. This property changes an individual tab item’s selected state. This is shown with a selection indicator (a gray-900 line) under or next to the selected tab item.
+ * 
+ * ## Orientation
+ * Tabs can be either horizontal or vertical. By default, tabs are horizontal and should be used when horizontal space is limited.
+ * 
+ * ## Density
+ * Tabs come in 2 densities: regular and compact. 
```

Comment:
```suggestion
/**
 * Tabs organize content into multiple sections and allow users to navigate between them. The content under the set of tabs should be related and form a coherent unit. Tabs can be either horizontal or vertical.
 * 
 * ## Usage notes
 *
 * ## Use icons consistently
 * Icons are optional, but don’t mix the use of icons in tabs if they are utilized. Navigation controls require a clear spacial relationship to one another, and mixing the use of icons can dramatically impact the visual balance and presence for each tab item.
 * 
 * ## Setting the selected tab item
 * Only one tab item can be selected at any given time. The selected tab item is designated by the `is-selected` class. A selection indicator line is shown under or next to the selected tab item.
```
Some suggestions:
- The headings were a bit big and needed a downgrade to a lower heading level under usage notes
- Moved the horizontal/vertical sentence to the description; it doesn't seem to need its own section. And the bit about only using vertical if there is enough horizontal space is already covered in the Vertical description. [slightly opinionated on this and the next one]
- Removed the density bit as it does not seem necessary, since regular is just the default. I could see maybe keeping it, but reorganizing it somewhere else.
- The selected tab item text was a little weird with the sentence about "property" and didn't mention the class used for selection. I don't think it's necessary or a good idea to reference the color of the line.

Reply:
Thanks, I took most of this! Is it looking better? https://github.com/adobe/spectrum-css/pull/2925/commits/0fde28dc9a8d3f9debcecfba5bc2075ede5a13af

I fixed up the headings and removed the density section. I did move the part about horizontal being default to the default story description. And thanks for reworking that selection indicator part- I wonder if "property" is specific to a toggle in Figma? You're totally right that it didn't really make sense.